azkaban-common/src/main/java/azkaban/event/EventData.java

```diff
@@ -15,20 +15,19 @@
    * Creates a new EventData instance.
    *
    * @param status node status.
+   * @param nestedId node id, corresponds to {@link ExecutableNode#getNestedId()}.
    */
-  public EventData(Status status) {
-    this(status, null);
+  public EventData(Status status, String nestedId) {
+    this.status = status;
+    this.nestedId = nestedId;
   }
 
   /**
    * Creates a new EventData instance.
-   *
-   * @param status node status.
-   * @param nestedId node id, corresponds to {@link ExecutableNode#getNestedId()}.
+   * @param node node.
    */
-  public EventData(Status status, String nestedId) {
-    this.status = status;
-    this.nestedId = nestedId;
+  public EventData(ExecutableNode node) {
```

Comment:
I'd like to avoid that bigger change. For this I can easily see that I'm not breaking anything.